packages/material-ui/src/AppBar/AppBar.d.ts

```diff
@@ -5,7 +5,7 @@ export interface AppBarProps extends StandardProps<PaperProps, AppBarClassKey> {
   /**
    * The color of the component. It supports those theme colors that make sense for this component.
```

Comment:
The description should probably be updated, since 'transparent' isn't a theme color.

Reply:
What new description do you have in mind?

Reply:
“It supports transparent and this theme colors that make sense ...”?

Reply:
I guess `inherit` has the same problem, and it hasn't caused any concern so far with this or other components, so I guess we can leave it as is for now.